src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
+  }
+
+  prefix(request) {
+    return request;
+  }
+
+  postfix(response) {
```

Comment:
I'm just saying that `onResponse` is potentially as confusing due to request/response existing outside of fetch. Although, I guess since this is fetch-based, it doesn't matter at all.